src/app/settings/SettingsPage.tsx

```diff
@@ -385,7 +385,7 @@ class SettingsPage extends React.Component<Props> {
                 ))}
             </section>
 
-            <WishListSettings />
+            <WishListSettings onSettingChange={this.onChange} />
```

Comment:
I'm quite unhappy with this being a thing, but I still felt it was better than replicating the onChange method inside WishListSettings, and keeps this change handler as a single place to update if the behaviour required to set settings ever changes.